src/createServer.js

```diff
@@ -1,3 +1,66 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => http.createServer((req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+
+  const textToConvert = String(req.url).split('?')[0].slice(1);
+  const toCase = new URLSearchParams(url.searchParams).get('toCase');
+
+  res.setHeader('Content-Type', 'application/json');
+
+  const errors = [];
+  const supportedCases = [
+    'SNAKE',
+    'KEBAB',
+    'CAMEL',
+    'PASCAL',
+    'UPPER',
+  ];
+
+  if (!textToConvert) {
+    errors.push({
+      message: 'Text to convert is required. '
```

Comment:
All error messages can be moved to variables with proper names, f.e. `noTextMessage`. So it can be reused in future.